packages/@react-spectrum/list/src/styles.css

```diff
@@ -340,6 +340,12 @@
     grid-area: content;
   }
 
+  &.react-spectrum-ListView--hasDescription {
```

Comment:
oo, wait, this is wrong, it should be `&:not(.react-spectrum-ListView--hasDescription)`